src/index.js

```diff
@@ -6,6 +6,7 @@ const defaults = {
   renderHost: 'https://{organization}.rokka.io',
   apiVersion: 1,
   transport: {
+    requestTimeout: 10000,
```

Comment:
Please set that timeout way higher. the request throtteling can easily let you wait for 20 seconds. So I'd say set this to 30 seconds